Header  header
bool    flag
uint32  ID_ROBOT
uint32  capacity
uint32  initial_vertex